media_driver/agnostic/gen11/codec/hal/codechal_huc_cmd_initializer_g11.h

```diff
@@ -199,6 +197,9 @@ class CodechalCmdInitializerG11 : public CodechalCmdInitializer
         PMOS_COMMAND_BUFFER cmdBuffer,
         HucCopyParams* params);
 
+#if defined (_HEVC_ENCODE_VME_SUPPORTED) || defined (_HEVC_ENCODE_VDENC_SUPPORTED)
+    virtual MOS_STATUS CmdInitializerSetDmem(bool brcEnabled) override;
```

Comment:
should the functions call this also check the hevc macro?

Reply:
These functions only used by hevc in codechal_vdenc_hevc_g11.cpp. So we don't need more hevc macro.
Thanks!

Reply:
codechal_huc_cmd_initializer_g11.cpp and codechal_huc_cmd_initializer.cpp have use hevc macro for these functions.